scss/pivotgrid/_layout.scss

```diff
@@ -0,0 +1,176 @@
+@include exports('pivotgrid/layout') {
+    .k-pivot {
+        position: relative;
+    }
+
+    .k-pivot-toolbar {
+        padding: $padding-x;
+        border-bottom-width: 1px;
+        border-bottom-style: solid;
+    }
+
+    .k-pivot-toolbar .k-button,
+    .k-fieldselector .k-list li.k-item {
+        @include border-radius( $border-radius );
+        padding: $button-padding-y ($button-padding-x * 7) $button-padding-y $button-padding-x;
+        //margin-right: $padding-x;
+        font-size: $font-size;
+        line-height: $form-line-height;
+        text-align: left;
+        position: relative;
+        cursor: move;
+        white-space: normal;
+    }
+
+    .k-fieldselector .k-list li.k-item {
+        margin: $button-padding-y-sm;
+    }
+
+    .k-settings-measures .k-button {
+        margin-bottom: $button-padding-y-sm;
+    }
+
+    .k-settings-columns .k-button {
+        margin-right: $button-padding-x;
+    }
+
+    .k-field-actions {
+        position: absolute;
+        right: $padding-y;
+        top: $padding-y;
+        line-height: 1;
+        cursor: pointer;
+    }
+
+    .k-pivot-layout {
+        border-spacing: 0;
+        table-layout: auto;
+    }
+
+    .k-pivot-layout > tbody,
+    .k-pivot td {
+        vertical-align: top;
+    }
+
+    .k-pivot-layout > tbody > tr > td {
+        padding: 0;
+    }
+
+    .k-pivot-rowheaders > .k-grid,
+    .k-pivot-table > .k-grid {
+        border-width: 0;
+    }
+
+    .k-pivot-rowheaders > .k-grid td:first-child,
+    .k-pivot-table .k-grid-header .k-header.k-first {
+        border-left-width: 1px;
+    }
+
+    .k-pivot-rowheaders > .k-grid td.k-first {
+        border-left-width: 0;
+    }
+
+    .k-pivot-rowheaders > .k-grid {
+        overflow: hidden;
+    }
+
+    .k-pivot-table {
+        border-left-width: 1px;
+        border-left-style: solid;
+    }
+
+    .k-pivot-table .k-grid-header-wrap > table {
+        height: 100%;
+    }
+
+    .k-pivot .k-grid-header .k-header {
+        vertical-align: top;
+    }
+
+    .k-pivot-layout .k-grid td {
+        border-bottom-width: 1px;
+    }
+
+    .k-pivot-layout .k-grid-footer > td {
+        border-top-width: 0;
+    }
+
+    .k-pivot-filter-window .k-treeview {
+        max-height: 600px;
+    }
+
+    /* selector */
+    .k-fieldselector .k-edit-buttons {
+        bottom: auto;
+    }
+
+    .k-fieldselector .k-edit-label {
```

Comment:
Consider nesting the `k-fieldselector` rules, it will make the file easier to read.